langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -215,6 +303,13 @@ public isolated function getContent(ProcessingInstruction|Comment x) returns str
 
 # Creates a new xml element item.
 #
+# ```ballerina
+# string name = "book";
+# map<string> attributes = {title: "Ballerina", author: "Anjana"};
+# xml contents = xml `<year>2022</year>`;
```

Comment:
Separate variables aren't required based on the new approach?

Reply:
Applicable to others `create` functions also.

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/8bceaabd7a1a2202d2929fee34d1e15e2f2b9b18